src/components/IdentityView/IdentityView.tsx

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react'
 import { connect } from 'react-redux'
 
+import { Link } from 'react-router-dom'
```

Comment:
Question:
Why this import?
Make sure your linter is catching these, it saves you time.

Reply:
Ok, it wasn't and still doesn't.